ci_tests.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -ex
+
+export MONTAGU_PROXY_TESTS=montagu-proxy-tests
+
+# If we do not have a node_modules dir (which will be the case on teamcity agent), create a volume so we can access
+# the node_modules installed during the tests to copy into the proxy image
+NODE_MODULES_DIR=$PWD/node_modules
```

Comment:
Mounting a path from the current working directory is a problem because Teamcity then can't clean them up: http://teamcity.montagu.dide.ic.ac.uk:8111/viewLog.html?buildId=37608&tab=buildResultsDiv&buildTypeId=montagu_reverseproxy_build

I think this approach would work with a named volume, or either way if the volume was cleaned up at the end. Alternatively, the docker in docker approach is maybe a bit less hacky!